src/main/java/uk/gov/companieshouse/api/accounts/service/ResourceService.java

```diff
@@ -16,12 +16,10 @@
         HttpServletRequest request)
         throws DataException;
 
-    ResponseObject<T> findById(String id, HttpServletRequest request)
+    ResponseObject<T> find(String companyAccountsId, HttpServletRequest request)
```

Comment:
If you're passing around `String`s representing `companyAccountsId`, you should be passing around the `CompanyAccounts` object instead and getting the ID where you need it

Reply:
The `CompanyAccount` REST object doesn't have an id exposed, what would be the suggested course of action?

Reply:
We can't use the `CompanyAccount` REST object if it doesn't have a relevant ID on it, so I'll approve this as the `companyAccountsId` `String` is the right thing to use.